packages/scoped-elements/src/ScopedElementsMixin.js

```diff
@@ -2,35 +2,44 @@
 import { TemplateResult } from 'lit-element';
 import { dedupeMixin } from '@open-wc/dedupe-mixin';
 import { transform } from './transform.js';
+import { defineLazyElement } from './registerElement.js';
 
 /**
  * @typedef {import('lit-html/lib/shady-render').ShadyRenderOptions} ShadyRenderOptions
  * @typedef {function(TemplateResult, Element|DocumentFragment|ShadowRoot, ShadyRenderOptions): void} RenderFunction
  */
 
 /**
- * Template cache
+ * Template caches
  *
  * @type {WeakMap<Function, Map<TemplateStringsArray, TemplateStringsArray>>}
  */
 const templateCaches = new WeakMap();
 
+/**
+ * Lambda caches
+ *
+ * @type {WeakMap<object, Map<string, function(): Promise<typeof HTMLElement>>>}
+ */
+const lambdaCaches = new WeakMap();
```

Comment:
I had to create this lambda cache because each time we call scopedElements new lambdas were created and I nedd to maintain a reference to link it with the scoped tag name.

Reply:
I suggest calling it functionCache :)

Reply:
I know lambdaCache was not a good name for JS world 😆 

functionCache it's ok for me 👍